yarn.lock

```diff
@@ -906,6 +906,13 @@
   dependencies:
     regenerator-runtime "^0.13.2"
 
+"@babel/runtime@^7.4.4", "@babel/runtime@^7.5.5", "@babel/runtime@^7.7.2", "@babel/runtime@^7.8.7":
```

Comment:
`yarn.lock` should be in your `.gitignore` file, and should not be committed. Can you please remove this file from this PR? Everything else looks good!